magda-web-client/src/Components/Dataset/View/DatasetPageSuggestForm.js

```diff
@@ -85,27 +85,36 @@ export default class DatasetPageSuggestForm extends React.Component {
         };
         const formProps = {
             title: false,
+            description:
+                "Your question will be sent to the publishers of the data: " +
+                this.props.contactPoint,
```

Comment:
could be worth replicating server side code here
i.e.

this.props.contactPoint.match(/([a-zA-Z0-9._-]+@[a-zA-Z0-9._-]+\.[a-zA-Z0-9._-]+)/g);

or 

```
function getEmailFromText( text ) {
  return text.match(/([a-zA-Z0-9._-]+@[a-zA-Z0-9._-]+\.[a-zA-Z0-9._-]+)/g);
}
```

Reply:
actually might be better to do without the /g, if we only want the first match

Reply:
Thanks! Updated it. included the /g in the end as without it there were multiple matches for the same email address, resulting in repeated outputs.